go/status/log_send.go

```diff
@@ -4,27 +4,21 @@
 package status
 
 import (
-	"archive/tar"
```

Comment:
sorry about the huge diff here. the helper functions were moved without change to `status/utils.go`